### PR TITLE
Our services section

### PR DESCRIPTION
**Pull request by Krafty-Coder**
# What changes have you made
  - Changed icon size and the icon themselves. 
  - Modified the content of the service section
  - Add general content to the service section
  - Decreased padding after the title
  - Changed the font-size of the h2 tag
  -  ~Changed the font-type as requested~
  - Put the default font in the services section
  - :stuck_out_tongue: 

# Screenshots of the changes
### Before

screenshot from 2018-04-17 10-17-24

### After

screenshot from 2018-04-17 12-57-14

### Before

phone-services-view gif

### After

peek 2018-04-17 13-12

# Checks
  - [x] my code is well formatted
  - [x] I only checked the required files

### CODE REVIEW

File: src/pages/index.html [hunk 1]
```diff
@@ -8,25 +8,29 @@ <h1>Who we are</h1>
...
+    <div class="medium-12">
+	  <p>We help brands who seek the full scope of a powerful digital presence. Podii in-house</p>
+	  <p>experts create web, social, and content experiences tailored to your needs in the modern technological era.</p>
+    </div>
     <div class="cell medium-4">
       <div class="service">
-        <img src="{{root}}/assets/img/coding.png" alt="" width="50">
-        <h2>Web development</h2>
-        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit.</p>
+        <img src="{{root}}/assets/img/web-app.png" alt="" width="128">
```
Comment (sigu) [inline src/pages/index.html]: Remove the leading `/` from the src, the src should be of the form `{{root}}assets/img/web-app.png` and not `{{root}}/assets/img/web-app.png`

File: src/pages/index.html [hunk 2]
```diff
@@ -8,25 +8,29 @@ <h1>Who we are</h1>
...
+        <h2>Web Applications</h2>
+        <p>We build best website applications and this is our expertise. Websites are just static views on the web. Web applications on the other hand allows users to create accounts, get confidential data, store data and do a lot of cool stuff on the web.</p>
       </div>
     </div>
     <div class="cell medium-4">
       <div class="service">
-        <img src="{{root}}/assets/img/coding.png" alt="" width="50">
-        <h2>Web development</h2>
-        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit.</p>
+        <img src="{{root}}/assets/img/web-dev.png" alt="" width="128">
```
Comment (sigu) [inline src/pages/index.html]: All images must have `alt` attribute filled in

File: src/pages/index.html [hunk 3]
```diff
@@ -51,7 +55,7 @@ <h5>Meetups</h5>
       <div class="media-item">
         <i class="fi fi-trophy" aria-hidden="true"></i>
         <h1>450</h1>
-        <h5>Cups & coffee</h5>
+        <h5>Cups of coffee</h5>
```
Comment (sigu) [inline src/pages/index.html]: This is out of the scope of this PR

File: src/pages/index.html [hunk 4]
```diff
@@ -8,25 +8,29 @@ <h1>Who we are</h1>
 <section id="services">
   <h1>Our Services</h1>
   <div class="grid-x grid-margin-x">
+    <div class="medium-12">
```
Comment (sigu) [inline src/pages/index.html]: Why do we have `medium-12` at this point? We are doing mobile first development

File: src/assets/scss/_index.scss [hunk 5]
```diff
@@ -8,15 +8,21 @@
   text-align: center;
   padding:  2rem ;
 }
-#services{
+#services {
   text-align: center;
   padding: 2rem;
   background-color: #e6e6e6;
+  .service-descr {
+      padding: 0;
```
Comment (sigu) [inline src/assets/scss/_index.scss]: use the team indentation level of 2 spaces, Avoid 4 spaces or tabs

**Comment by GettyOrawo** [comment]
In my opinion the words after the 'Services' Heading are unnecessary. We could opt for just having the heading then the three services immediately after. 
...maybe different opinions from others... :)

File: src/assets/scss/_index.scss [hunk 6]
```diff
@@ -7,22 +7,29 @@
   text-align: center;
   padding:  2rem ;
 }
-#services{
+#services {
+  font-family: 'Slabo 27px', serif;
```
Comment (champagnepappi) [inline src/assets/scss/_index.scss]: use 'ems' instead of pixels for font size

Comment (Lmise) [reply src/assets/scss/_index.scss]: Good job.

**Comment by champagnepappi** [comment]
I feel the font-size for the services description is smaller

**Comment by Lmise** [comment]
check on the mobile view.The title  are not bold enough(its almost the same as the  rests of the text)

**Comment by pomochieng** [comment]
View the titles on the mobile view After corrections, You are not able to differentiate them with  the content  text.

**Comment by Krafty-Coder** [comment]
The service section on mobile will be solved by the issue on colors which is issue #22

**Pull request merged**